### PR TITLE
Add malmitalo and fix back button functionaliy on event page

### PR DESCRIPTION
**Pull request by Sartonon**
## Description :sparkles:

- Add malmitalo and fix back button functionality on event page

## Issues :bug:
**[TH-902](https://helsinkisolutionoffice.atlassian.net/browse/TH-902):**

### CODE REVIEW

File: src/domain/event/eventCard/EventCard.tsx [hunk 1]
```diff
@@ -30,6 +31,7 @@ const EventCard: React.FC<Props> = ({ event }) => {
   const history = useHistory();
   const { search } = useLocation();
   const { t } = useTranslation();
+  const params = useParams<{ place?: string }>();
```
Comment (jorilindell) [inline src/domain/event/eventCard/EventCard.tsx:34]: Could be useful to mention here why place is got from the url parameter. It's not 100% clear that place is used only on the place specific paces. Can be little bit confusing to new developer in the team.

File: src/domain/event/eventCard/LargeEventCard.tsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import classNames from 'classnames';
 import { Button } from 'hds-react';
```
Comment (jorilindell) [inline src/domain/event/eventCard/LargeEventCard.tsx:2]: This is not related to this PR. But there are quite much same logic on  EventCard and LargeEventCard. The layout and buy button are pretty much only differences. I know this was my decision but it could be good idea to have all the logic in single component. Could be little bit easier to maintain.

File: src/domain/event/eventCard/LargeEventCard.tsx [hunk 3]
```diff
@@ -30,6 +31,7 @@ interface Props {
 const LargeEventCard: React.FC<Props> = ({ event }) => {
   const { t } = useTranslation();
   const { push } = useHistory();
+  const params = useParams<{ place?: string }>();
```
Comment (jorilindell) [inline src/domain/event/eventCard/LargeEventCard.tsx:34]: Could be useful to mention here why place is got from the url parameter. It's not 100% clear that place is used only on the place specific paces. Can be little bit confusing to new developer in the team.

**Review by jorilindell — APPROVED** [review]
Looks good. Added some quite minor improvement ideas.

File: src/domain/event/eventCard/EventCard.tsx (hunk 1 above)
Comment (Sartonon) [reply src/domain/event/eventCard/EventCard.tsx:34]: true
